packages/spear-cli/src/plugins/spear-seo.ts

```diff
@@ -0,0 +1,102 @@
+import { parse } from "node-html-parser"
+import { loadFile } from "../util.js"
+import { Component, Element, HookApi, SpearSettings, SpearState } from "../interfaces/HookCallback"
+
+let globalSettings: Map<string, string>
+
+// This plugin will replace `spear-seo` tag.
+// SettingsFile:
+//   JavaScript Object files which has key and value like following.
+//   export default {
+//     title: "Blog site.",
+//   }
+export function spearSEO(settingsFile?: string): HookApi {
+    // Use configuration and afterBuild hook for generating SEO.
+    return {

```

Comment:
It may be better to use ESLint because there is an indent deviation. 🙏 
(when we have time)
- #107